projects/templates/src/lib/components/po-page-dynamic-search/po-page-dynamic-search.component.ts

```diff
@@ -139,6 +165,22 @@ export class PoPageDynamicSearchComponent extends PoPageDynamicSearchBaseCompone
     return new Date(year, month - 1, day).toLocaleDateString(getBrowserLanguage());
   }
 
+  private formatsFilterValuesToUpdateDisclaimers(filters) {
+    const formattedFilters = filters.reduce(
+      (result, item) =>
+        Object.assign(result, { [item.property]: item.value ? item.value : item.initValue || undefined }),
```

Comment:
```suggestion
        Object.assign(result, { [item.property]: item.value || item.initValue }),
```

Csao não existir o initValue será atribuido undefined, conforme a logica anterior